pkg/kubefed/init/init.go

```diff
@@ -459,7 +461,12 @@ func (i *initFederation) Run(cmdOut io.Writer, config util.AdminConfig) error {
 }
 
 func createNamespace(clientset client.Interface, federationName, namespace string, dryRun bool) (*api.Namespace, error) {
-	ns := &api.Namespace{
+	ns, err := clientset.Core().Namespaces().Get(namespace, metav1.GetOptions{})
```

Comment:
Reasons for this change? I think kubefed currently assumes that if the namespace already exists there is something wrong and fails. I think its better to keep it that way.

Reply:
Whether we should allow deploying fcp in existing namespace is debatable. Please refer to this issue https://github.com/kubernetes/federation/issues/19, where many people are expressing the need to allow such a thing. Also there is no strong case on why we should not allow. it is easy for the user to delete the federation namespace if the initialisation fails, and continue deploying further. Let me know your opinion and i will change accordingly. Thanks !

Reply:
Makes sense, please keep it as is; or rather submit another PR for the namespace commit alone as a resolution to https://github.com/kubernetes/federation/issues/19. Thanks!

Reply:
@irfanurrehman, Done changes as suggested. PTAL